src/shared/sandboxRequest.ts

```diff
@@ -143,11 +149,68 @@ export async function getSrcIdByName(conn: Connection, sandboxName: string): Pro
   }
 }
 
+export async function querySandboxProcessFeatures(
+  conn: Connection,
+  identifier: string,
+  isId: boolean
+): Promise<string | undefined> {
+  const query = isId
+    ? `SELECT Features FROM SandboxProcess WHERE Id = '${identifier}'`
+    : `SELECT Features FROM SandboxProcess WHERE SandboxName = '${identifier}'`;
+
+  try {
+    const result = await conn.singleRecordQuery<{ Features: string }>(query, { tooling: true });
+    return result.Features;
+  } catch (err) {
+    throw cloneMessages.createError('error.featuresQueryFailed', [identifier], [], err as Error);
+  }
+}
```

Comment:
The Org class already has various `querySandboxProcess()` methods so I'd rather use that code.